templates/generation/generate_release_please_config.sh

```diff
@@ -15,7 +15,7 @@ rp_manifest_line=""
 
 for module in $(find . -mindepth 2 -maxdepth 2 -name pom.xml |sort | xargs dirname); do
 
-  if [[ "${module}" = *google-cloud-gapic-bom ]] || [[ "${module}" = *CoverageAggregator ]]; then
+  if [[ "${module}" = *google-cloud-gapic-bom ]] || [[ "${module}" = *CoverageAggregator ]] || grep -q 'SNAPSHOT<\/version>' "${module}/pom.xml" ; then
```

Comment:
skip the whole versions update if it's already on `SNAPSHOT`.

Reply:
I've modified this script since this commit. The latest version of it can be found here -> https://github.com/googleapis/google-cloud-java/blob/main/generation/generate_release_please_config.sh
We can just run this one on the main branch and generate the configs.

Reply:
The version in `main` looks the same to me.